templates/download/index.twig

```diff
@@ -7,18 +7,18 @@
 		   href="{{ rel.browser_download_url }}">
 			<span id="button-title">LMMS</span><br>
 			<span class="fa fa-download download-icon"></span>
-			<big>Free Download</big><br>
+			<big>{% trans %}Free Download{% endtrans %}</big><br>
 			<small>{{ rel.release.name }}&nbsp;({{ rel.osname }})</small>
 		</a>
 	{% endif %}
 {% endmacro %}
 
 {% macro releasenotes(notes, divid) %}
 <div id="release-notes">
-	<a data-toggle="collapse" href="#collapse_{{ divid }}">Show release notes<br><i class="fa fa-angle-down"></i></a>
+	<a data-toggle="collapse" href="#collapse_{{ divid }}">{% trans %}Show release notes{% endtrans %}<br><i class="fa fa-angle-down"></i></a>
 	<div id="collapse_{{ divid }}" class="panel-collapse collapse">
 		<div class="release-notes well">
-			{{ notes|markdown }}
+			{{ ('|GH|' ~ notes)|markdown }}
```

Comment:
Because I want to render GitHub Markdown (release note) using GitHub API and render our other content using offline parsedown render, I added a flag to indicate this will be rendered by GitHub render, and since we now have a offline render, when GitHub API is not available, it will automatically fallback to offline render.